### ISSUE TITLE
Docker image is built for ARM machines, most linux environments are Intel

### ISSUE THREAD

**Issue by alishobeiri** [WhatsApp/proxy#116]
I believe the premade docker image was made on a M1 mac and therefore is configured only for armv8 platforms. 

Would it be possible to have one version supporting Intel machines as well so it can be easy to use with providers such as AWS?

**Comment by alishobeiri** [comment]
@slawlor Just so you're aware, also very cool to see a fellow McGill grad made this, very cool stuff :)

**Comment by slawlor** [comment]
new M1, who dis? /s

Thanks for reporting this! I think I've pushed a new image which is cross-built, would you mind checking? I have not yet landed our CI pipeline (#114) which should run on linux (always) but i'll add platform specification to it to be sure it cross-builds as well!

**Comment by nghazali** [comment]
I just used the built image, and this is built for linux/arm64 as is mentioned above. However, in order to run on intel machine, when I use `--platform linux/arm64` ro run the docker, still I get this error message:

```
wa:~# docker run -it --platform linux/arm64 -p 80:80 -p 443:443 -p 5222:5222 facebook/whatsapp_proxy:latest
exec /usr/local/bin/docker-entrypoint.sh: exec format error
```
